core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -927,7 +928,14 @@ class CivilizationInfo : IsPartOfGameInfoSerialization {
         for (city in cities) city.startTurn()  // Most expensive part of startTurn
 
         for (unit in getCivUnits()) unit.startTurn()
-        hasMovedAutomatedUnits = false
+
+        if (playerType == PlayerType.Human && UncivGame.Current.settings.automatedUnitsMoveOnTurnStart) {
+            hasMovedAutomatedUnits = true
+            Concurrency.run("Move automated units") {
+                for (unit in getCivUnits())
```

Comment:
It's not good to run this in another thread, since the user could manually move units and we'll end up with a conflict

Reply:
Do you propose remove threading here or rather add user input blocking measures?

Reply:
Remove threading

Reply:
Done